src/useRouteQueryParams/useRouteQueryParams.ts

```diff
@@ -0,0 +1,65 @@
+import { Ref } from 'vue'
+import { NoInfer } from '@/types/generics'
+import { isRouteParamClass, RouteParamClass } from '@/useRouteQueryParam/formats'
+import { useRouteQueryParam } from '@/useRouteQueryParam/useRouteQueryParam'
+
+type AnyRecord = Record<string, unknown>
```

Comment:
probably more like `UnknownRecord`, or `UnknownObject`. We really should export types like this somewhere

Reply:
Yeah this is a dumb type so I could be lazy. Its really not even that accurate. `Record<string | number | symbol, unknown>` would be a better `AnyRecord` type. But I don't want to check for `number` or `symbol` keys.